store/servers.ts

```diff
@@ -56,16 +68,15 @@ export const actions: ActionTree<ServerState, ServerState> = {
               address: serverUrl
             });
           }
-
-          this.$router.push('/login');
+          if (!silent) this.$router.push('/login');
         }
-      } else {
+      } else if (!silent) {
         dispatch('notifyServerVersionIsLow');
       }
     } catch (err) {
       // eslint-disable-next-line no-console
       console.error(err); // in case something inside the try rather than a request failure
-      dispatch('notifyServerCantBeFound');
+      if (!silent) dispatch('notifyServerCantBeFound');
```

Comment:
This one is inspired from spending significant amount of time to understand that at some point I got `server not found` error. Bringing back a `silent` variation is just getting back to the situation something spending a lot of time to understand some code suppressed an important message.

Reply:
Maybe if you do a middleware to listen for that action and if it was fired then execute another one to select localhost.

Reply:
The reason the error is supressed, is that the user might not want to connect to `localhost:8096`, or a server might not exist there, we are only checking it as a matter of convenience. If they attempt to type in the address manually, and connect the error will still show.

Reply:
If the user might not want to connect, why do we force it in the first place?

Reply:
Purely as a check for convenience. In the future, the Address that is checked will be specified in a config.Json file